app/controllers/bands_controller.rb

```diff
@@ -1,10 +1,46 @@
 class BandsController < InheritedResources::Base
 
+  before_action :find_band, except: [:index, :new, :create]
```

Comment:
Because this
```
only: [:show, :edit, :update, :destroy]
```
Is the exact same thing as this
```
except: [:index, :new, :create]
```
With the exception that the latter requires me (or anybody) to type less.